### PR TITLE
[risk=low][no ticket] Bump all e2ev2 packages to latest, to resolve a vulnerability

### PR DESCRIPTION
**Pull request by jmthibault79**
Resolves https://github.com/all-of-us/workbench/security/dependabot/88

---
**PR checklist**

- [ ] I have included an issue ID or "no ticket" in the PR title as outlined in [CONTRIBUTING.md](https://github.com/all-of-us/workbench/blob/main/.github/CONTRIBUTING.md).
- [ ] I have included a risk tag of the form `[risk=no|low|moderate|severe]` in the PR title as outlined in [CONTRIBUTING.md](https://github.com/all-of-us/workbench/blob/main/.github/CONTRIBUTING.md).
- [ ] I have manually tested this change and my testing process is described above.
- [ ] This change includes appropriate automated tests, and I have documented any behavior that cannot be tested with code.
- [ ] I have added explanatory comments where the logic is not obvious.
- One or more of the following is true:
  - [ ] This change is intended to complete a JIRA story, so I have checked that all AC are met for that story.
  - [ ] This change fixes a bug, so I have ensured the steps to reproduce are in the Jira ticket or provided above.
  - [ ] This change impacts deployment safety (e.g. removing/altering APIs which are in use), so I have documented the impacts in the description.
  - [ ] This change includes a new feature flag, so I have created and linked new JIRA tickets to (a) turn on the feature flag and (b) remove it later.
  - [ ] This change modifies the UI, so I have taken screenshots or recordings of the new behavior and notified the PO and UX designer in [Slack](https://pmi-engteam.slack.com/archives/C02MWP2RN5P).
  - [ ] This change modifies API behavior, so I have run the relevant E2E tests locally because API changes are not covered by our PR checks.
  - [ ] None of the above apply to this change.

### CODE REVIEW

File: e2ev2/src/utils.js [hunk 1]
```diff
@@ -111,6 +111,6 @@ export_({denseDateTime})
 // Currently, we don't mock Leo requests so the setCookie request always fails. This hack dismisses the error modal
 // and should be removed when we mock Leo requests.
 const dismissLeoAuthErrorModal = (page) => {
-  return page.waitForXPath("//div[@role='dialog']//div[@role='button'][contains(., 'OK')]").then(b => b.click());
```
Comment (jmthibault79) [inline e2ev2/src/utils.js:114]: removed in puppeteer-core 22

File: ui/src/environments/environment-type.ts [hunk 2]
```diff
@@ -71,6 +71,7 @@ export interface Environment extends EnvironmentBase {
   // Indicates that the current server is a local server where client-side
   // debugging should be enabled (e.g. console.log, or devtools APIs).
   debug: boolean;
+
```
Comment (jmthibault79) [inline ui/src/environments/environment-type.ts:74]: trivial UI change to trigger tests

**Review by evrii — APPROVED** [review]
The changes look good to me. I was not able to get the tests to pass locally, but it does not seem related to your changes.

**Comment by jmthibault79** [comment]
@evrii I was able to run tests locally.  All passed except for a timeout on create-user, which is also what I see on the main branch.

**Pull request merged**